event-discord-google_calender/main.py

```diff
@@ -0,0 +1,12 @@
+# SCRIPT
+
+# Execute event_details.py
+import event_details
+
+# Execute discord_.py
+import discord_
+
+# Execute google_calender.py
+import google_calender_
+
+# END
```

Comment:
Club this file with event_details.py script, import everything in main.py, execute the process